packages/hooks-react/src/useProtectedMedia.ts

```diff
@@ -2,15 +2,22 @@ import { useQuery } from 'react-query';
 import type { PlaylistItem } from '@jwp/ott-common/types/playlist';
 import ApiService from '@jwp/ott-common/src/services/ApiService';
 import { getModule } from '@jwp/ott-common/src/modules/container';
+import { useTranslation } from 'react-i18next';
 
 import useContentProtection from './useContentProtection';
 
 export default function useProtectedMedia(item: PlaylistItem) {
   const apiService = getModule(ApiService);
-  const contentProtectionQuery = useContentProtection('media', item.mediaid, (token, drmPolicyId) => apiService.getMediaById(item.mediaid, token, drmPolicyId));
+  const contentProtectionQuery = useContentProtection('media', item.mediaid, (token, drmPolicyId) =>
+    apiService.getMediaById(item.mediaid, language, token, drmPolicyId),
+  );
+
+  // Determine currently selected language
+  const { i18n } = useTranslation('menu');
+  const language = i18n.language;
 
   const { isLoading, data: isGeoBlocked } = useQuery(
-    ['media', 'geo', item.mediaid],
+    ['media', 'geo', item.mediaid, language],
```

Comment:
I don't think language is needed here. The GEO country doesn't change, so we don't want to revalidate GEO blocking...